.golangci.yml

```
@@ -48,8 +48,15 @@ linters:
   - misspell
   - whitespace
   - revive
+  - depguard
 
 linters-settings:
+  depguard:
+    rules:
+      main:
+        deny:
+        - pkg: "io/ioutil"
+          desc: "Use corresponding 'os' or 'io' functions instead."
```

```suggestion
          desc: "replaced by io and os packages since Go 1.16: https://tip.golang.org/doc/go1.16#ioutil"
```
How about this description? This would be more helpful for developers to fix lint warnings.
Copied from https://github.com/OpenPeeDeeP/depguard?tab=readme-ov-file#golangci-lint.

---

Great.